lms/envs/common.py

```diff
@@ -349,6 +349,9 @@
     'ENABLE_COURSEWARE_SEARCH': False,
 }
 
+# Add Theme values here
+THEME_VALUES = {}
```

Comment:
To allow customization from config/secrets, we'll need to attempt pulling this from `lms.env.json` first, but defaulting to `{}` when it doesn't exist.